CareKit/CareCard/OCKCareCardViewController.m

```diff
@@ -474,6 +474,20 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 }
 
 
+- (void)tableView:(UITableView *)tableView commitEditingStyle:(UITableViewCellEditingStyle)editingStyle forRowAtIndexPath:(NSIndexPath *)indexPath {
+    OCKCarePlanActivity *selectedActivity = _events[indexPath.row].firstObject.activity;
+    OCKCarePlanEvent *event = _events[indexPath.row].firstObject;
+
+    [self.store setEndDate:event.date
+               forActivity:selectedActivity
+                completion:^(BOOL success, OCKCarePlanActivity * _Nullable activity, NSError * _Nullable error) {
+                }];
```

Comment:
Use completion block to make sure the end date was set successfully. If not, maybe log it, or show an alert to the user?